skins/laika/tests/unit/components/shared/Email.spec.ts

```diff
@@ -54,8 +54,8 @@ describe( 'Email', () => {
 		const hasError = wrapper.vm.$data.emailHasError;
 		expect( hasError ).toBe( false );
 	} );
-
-	it( 'sends email to store if it is has valid format', () => {
+	//Move this test to the file with unit tests for the Address component once it exists
+	it.skip( 'sends email to store if it is has valid format', () => {
```

Comment:
Maybe add a test that checks if the event is emitted on blur?

Reply:
@gbirke Merging this even with your comment here because it's out of scope for this PR I would say. This way I can rebase this into my branch.

Reply:
😅 I think it was in the scope of the PR because I changed `Email.vue` in a way that requires the test Gabriel mentioned. But I'll add it in the next PR.

Reply:
@tzhelyazkova Sorry, didn't mean to merge this early, I must have misread this TODO. Either way, it's rebased into https://github.com/wmde/FundraisingFrontend/pull/1533 now and I am just wrapping that up.